src/bundler.ts

```diff
@@ -125,12 +125,41 @@ class Bundler {
     return false;
   }
 
-  hide(node) {
+  getHiddenNode(): ASTNode {
```

Comment:
Done. Added a TODO for renaming method as well to something more intention-revealing.